src/main/java/io/kestra/plugin/couchbase/Query.java

```diff
@@ -0,0 +1,136 @@
+package io.kestra.plugin.couchbase;
+
+import com.couchbase.client.java.Cluster;
+import com.couchbase.client.java.codec.TypeRef;
+import com.couchbase.client.java.json.JsonArray;
+import com.couchbase.client.java.json.JsonObject;
+import com.couchbase.client.java.query.QueryOptions;
+import com.couchbase.client.java.query.QueryResult;
+import io.kestra.core.models.annotations.Example;
+import io.kestra.core.models.annotations.Plugin;
+import io.kestra.core.models.tasks.RunnableTask;
+import io.kestra.core.runners.RunContext;
+import io.kestra.core.serializers.FileSerde;
+import io.kestra.plugin.couchbase.models.FetchType;
+import io.swagger.v3.oas.annotations.media.Schema;
+import lombok.*;
+import lombok.experimental.SuperBuilder;
+
+import javax.validation.constraints.NotBlank;
+import javax.validation.constraints.NotNull;
+import java.io.*;
+import java.net.URI;
+import java.util.List;
+import java.util.Map;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Schema(
+    title = "Query a Couchbase database with N1QL."
+)
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send a N1QL query to a Couchbase Database",
+            code = {
+                "connectionString: couchbase://localhost",
+                "username: couchbase_user",
+                "password: couchbase_passwd",
+                "query: SELECT * FROM COUCHBASE_BUCKET(.COUCHBASE_SCOPE.COUCHBASE_COLLECTION)",
+                "fetchType: FETCH"
```

Comment:
Please use STORE as the default fetchType as it's the best option

Reply:
Only in example or also within the Builder.Default ?